src/main/resources/application.yml

```
@@ -29,5 +29,7 @@ kakao:
 
 jwt:
   secret: ${JWT_SECRET}
-  access-token-validity: 1800000
-  refresh-token-validity: 604800000
\ No newline at end of file
+  access-token-validity: 1800000000
+  refresh-token-validity: 604800000
+domain:
+  base-url: ${DOMAIN_BASE_URL}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)